README.md

```diff
@@ -1,68 +1,81 @@
 # OpenTermsArchive.org
 
-WORK IN PROGRESS
+This is the repository for `opentermsarchive.org`, the website that presents the [Open Terms Archive](https://github.com/ambanum/OpenTermsArchive) project.
 
-## Development
+## Introduction
 
-Copy `.env.local.example` to `.env.local` file at the root of the project
+Build on [Next.js](https://nextjs.org) react framework, using [TypeScript](https://www.typescriptlang.org/) and [PostCSS](https://postcss.org/).
 
-```
-NODE_PATH="src"
-```
+## Configuration
 
-Then launch
+Copy `.env.local.example` to `.env.local` file at the root of the project and fill in the values of the constants listed below.
 
 ```
-yarn
-yarn dev
+NODE_PATH="src"
 ```
 
-### Test to your local IP address
+### Contribution interface
 
-When building responsive applications, it is very important to be able to test on mobile and other devices in real-time.
+The contribution interface, located at `/contribute` will help the user to create new services easily.
+
+Clicking on `Validate` can automatically create an issue un a github repository.
 
-Get your local IP
+For this, you need to setup some constants in your `.env.local`
 
 ```
-ipconfig getifaddr en0
+GITHUB_TOKEN_CREATE_ISSUE= #create one with repo privileges here https://github.com/settings/tokens
+GITHUB_OTA_OWNER=ambanum
+GITHUB_OTA_REPO=test-repo# or OpenTermsArchive
```

Comment:
Avoid `test-repo`, directly `GITHUB_OTA_REPO=OpenTermsArchive`

Reply:
In fact i will change this to the same env variables as on OTA Core so you can leave it